pycbc/psd/analytical_space.py

```diff
@@ -139,36 +342,41 @@ def analytical_psd_lisa_tdi_1p5_XYZ(length, delta_f, low_freq_cutoff,
     low_freq_cutoff : float
         Low-frequency cutoff for output FrequencySeries.
     len_arm : float
-        The arm length of LISA, in the unit of "m".
-    acc_noise_level : float
-        The level of acceleration noise.
-    oms_noise_level : float
-        The level of OMS noise.
+        The arm length of the detector, in the unit of "m".
+    psd_components : numpy.array
+        The PSDs of acceleration noise and OMS noise.
+    tdi : str
+        The version of TDI. Choose from "1.5" or "2.0".
 
     Returns
     -------
     fseries : FrequencySeries
-        The TDI-1.5 PSD (X,Y,Z channel) for LISA.
+        The TDI-1.5/2.0 PSD (X,Y,Z channel).
     Notes
     -----
-        Please see Eq.(19) in <LISA-LCST-SGS-TN-001> for more details.
+        Please see Eq.(19-20) in <LISA-LCST-SGS-TN-001> for more details.
     """
     len_arm = np.float64(len_arm)
-    acc_noise_level = np.float64(acc_noise_level)
-    oms_noise_level = np.float64(oms_noise_level)
     fr = np.linspace(low_freq_cutoff, (length-1)*2*delta_f, length)
-    s_acc_nu, s_oms_nu = lisa_psd_components(
-                          fr, acc_noise_level, oms_noise_level)
+    s_acc_nu = psd_components[0]
+    s_oms_nu = psd_components[1]
     omega_len = omega_length(fr, len_arm)
-    psd = 16*(np.sin(omega_len))**2 * (s_oms_nu+s_acc_nu*(3+np.cos(omega_len)))
+    psd = 16*(np.sin(omega_len))**2 * (s_oms_nu +
+                                       s_acc_nu*(3+np.cos(2*omega_len)))
+    if str(tdi) not in ["1.5", "2.0"]:
+        raise Exception("The version of TDI, currently only for 1.5 or 2.0.")
```

Comment:
I agree with Alex that make these Exceptions more specific, i.e. ValueError.

Reply:
Thanks, I have replaced all `raise Exception` with `raise ValueError`.